LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016-2017 Maxime Veber
```

Comment:
https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/LICENSE : they dit it :/